Fix the docs build on read the docs (#17)
tataratat/releso: README.md

# Reinforcement Learning based Shape Optimization (ReLeSO)

[![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/tataratat/releso/pypi_upload)](https://github.com/tataratat/releso)
[![Read the docs](https://readthedocs.org/projects/releso/badge/?version=latest)](https://releso.readthedocs.io/en/latest/?badge=latest)
[![PyPI - Version](https://img.shields.io/pypi/v/releso)](https://pypi.org/project/releso/)
[![Python Versions](https://img.shields.io/pypi/pyversions/releso)](https://pypi.org/project/releso/)
[![PyPI - License](https://img.shields.io/pypi/l/releso)](https://github.com/tataratat/releso/blob/main/LICENSE)

Releso is a Library/Framework for
Reinforcement Learning based Shape Optimization. Please look into the
Documentation for information on how it works. The instruction on how the
documentation can be built is given below as well as the instruction on how the

> package can be installed. But it is also available to install via `pip install releso`.

Reviewer: ```suggestion
package can be installed.  
Alternatively, it can be installed from `pip` via `pip install releso`.
```
